README.md

```diff
@@ -212,6 +212,107 @@ You may need to set up multiple automated transfer instances, for example if req
 
 In case different hooks are required for each instance, a possible approach is to checkout a new instance of the automation tools, for example in `/usr/lib/archivematica/automation-tools-2`
 
+Archivematica Client
+--------------------
+
+The transfers/amclient.py script is a module and CLI that provides functionality for interacting with the various Archivematica APIs.
+
+Basic usage: amclient.py <subcommand> [optional arguments] <positional argument(s)>
+  E.g.: amclient.py close-completed-transfers --am-user-name islandora 234deffdf89d887a7023546e6bc0031167cedf6
+  
+### Subcommands and arguments
+
+* close-completed-transfers
+  * purpose: close all completed transfers (those not failed or rejected)
+  * positional argument: am_api_key - API key for Archivematica dashboard user
+  * optional arguments: 
+    * --am-user-name <username> - username for Archivematica dashboard user (default: test)
+    * --am-url <url> - Archivematica URL (default: http://127.0.0.1)
+
```

Comment:
Can you modify all of these items so that the code and defaults are all back-tick-formatted. So "close-completed-transfers" should be \`close-completed-transfers\`, "am_api_key" should be \`am_api_key\`, "--am-user-name &lt;username&gt;" should be \`--am-user-name &lt;username&gt;\` and "--am-url &lt;url&gt;" should be \`--am-url &lt;url&gt;\`. Similarly for the defaults "test" and "http://127.0.0.1". Note again that the angle brackets around "&lt;username&gt;" are making that string invisible in the rendered markdown (which will be solved by the backticks).